### ISSUE TITLE
$edit link not being generated in admin default_body.php

### ISSUE THREAD

**Issue by cpaschen** [vdm-io/Joomla-Component-Builder#328]
### Steps to reproduce the issue

Trying to create a front-end edit link for an item.
Following the instructions in the tutorial video 

at approx 37:50

It says to go to the administrator/components/COMPONENT/views/LIST_VIEW/tmpl/default_body.php

### Expected result

Near the top it should have the edit variable/link.

i.e. 
$edit = "index.php?option=com_hello_world&view=greetings&task=greeting.edit";

### Actual result

That line is no longer anywhere in that file.
This is true for my own compiled files as well as the Hello World package.

### System information (as much as possible)
- OS Name & Version:  WAMP
- MySql Version: 
- Apache Version: 
- PHP Version: 
- Joomla Version: 
- JCB Version: 2.8.5
- Browser: 

### Additional comments

It appears that the compiler is no longer including this $edit variable with the edit link.
This is true for 2.8.5 and 2.8.4 in my testing.

Are there new instructions on how to create the edit link for the front-end?

**Comment by Llewellynvdm** [comment]
I just opened my hello world files on my computer, after running the compiler and install and this is what I see:
![image](https://user-images.githubusercontent.com/5607939/43556384-d111e8f0-95ff-11e8-8d49-7638e55496ef.png)

Can you give me a screen shot? Remember this is in the administrator area.

**Comment by Llewellynvdm** [comment]
Please check that all the needed PHP modules are active. Why not use Docker instead of WAMP... or even the Bitnami Joomla deployment options works very well.

I made this [quick tutorial](https://youtu.be/zpS52k89YcI) on how to use Bitnami with JCB, it just takes care of all the trouble of setup a local development environment, check it out.

**Comment by cpaschen** [comment]
Is there ANYTHING that you haven't created a tutorial for yet? :-)
That is a GREAT tutorial. 
I've tested using the Bitnami package and everything is working properly.
I've tested on a different machine with WAMP and it works fine so there is obviously some issue with this WAMP install that is at least contributing to the issue.
I'm closing this as it appears to be isolated to the environment and not an actual JCB bug.
